.env.template

```diff
@@ -5,6 +5,7 @@ FLASK_ENV=development
 # Change the following values for local development.
 TRELLO_API_KEY=
 TRELLO_API_TOKEN=
+TRELLO_API_BASE_URL=
```

Comment:
You've added this here and to the pipeline, but it's not actually used by your code anywhere, right?
You still do `self.trello_web_url = 'https://api.trello.com/1'`.

You should either update your code to actually use it, or remove all references to TRELLO_API_BASE_URL